x/distribution/types/delegator.go

```diff
@@ -4,19 +4,6 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-// starting info for a delegator reward period
-// tracks the previous validator period, the delegation's amount
-// of staking token, and the creation height (to check later on
-// if any slashes have occurred)
-// NOTE that even though validators are slashed to whole staking tokens, the
-// delegators within the validator may be left with less than a full token,
-// thus sdk.Dec is used
-type DelegatorStartingInfo struct {
```

Comment:
We must be very very careful about the JSON tags when migrating types to proto. The JSON tag for `height` is `creation_height` yet in the new proto type it's `height`.

I went through all the types and verified -- it was only this single field 👍